.changeset/quick-pillows-jump.md

```diff
@@ -0,0 +1,15 @@
+---
+"@fluidframework/tree": minor
+"fluid-framework": minor
+"__section": tree
```

Comment:
```suggestion
"__section": tree
"__highlight": true
```

Seems worth highlighting in the tree notes. Your call though. I think this only changes ordering and adds an icon or something to make it stand out in the notes.

Reply:
No existing code is impacted by this, and the API is only alpha so even people who it applies to likely should adopt it yet. So I don't think we want to highlight it.